packages/block-editor/src/components/grid-visualizer/grid-item-resizer.js

```diff
@@ -12,14 +13,89 @@ import { getComputedCSS } from './utils';
 
 export function GridItemResizer( { clientId, onChange } ) {
 	const blockElement = useBlockElement( clientId );
+	const rootBlockElement = blockElement?.parentElement;
+	const [ resizeDirection, setResizeDirection ] = useState( null );
+
+	/*
+	 * Resizer dummy is an empty div that exists only so we can
+	 * get the bounding client rect of the resizer element. This is
+	 * necessary because the resizer exists outside of the iframe, so
+	 * its bounding client rect isn't the same as the block element's.
+	 */
+	const resizerDummyRef = useRef( null );
+
 	if ( ! blockElement ) {
 		return null;
 	}
+
+	const justification = {
+		right: 'flex-start',
+		left: 'flex-end',
+	};
+
+	const alignment = {
+		top: 'flex-end',
+		bottom: 'flex-start',
+	};
+
+	const styles = {
```

Comment:
I mean we could do that but is there any advantage to it? Using `styles` is more consistent with how the component works, like with the `width` and `height` props.

Reply:
It lets us avoid adding the new `additionalStyles` prop. Not a big deal since `BlockPopoverCover` is not a public component but nice to keep API surface area low where possible.

Reply:
Hmm yeah. I think the additional prop is a worthwhile trade-off for better legibility. Also the inline style means it's less likely some over-specific CSS will break this at any point 😄